data/schema/v1/Decision_Point_Group-1-0-1.schema.json

```diff
@@ -16,16 +16,19 @@
                     "$ref": "#/$defs/schemaVersion"
                 },
 		"version": {
-		    "type": "string",
-		    "description": "Version (a semantic version string) that identifies this object"
+		    "$ref": "https://certcc.github.io/SSVC/data/schema/v1/Decision_Point-1-0-1.schema.json#/$defs/decision_point/properties/version"
 		},
 		"name": {
 		    "type": "string",
-		    "description": "A short label that captures the description of the Decision Point or the Group of Decision Points."
+		    "description": "A short label that captures the description of the Decision Point or the Group of Decision Points.",
+		    "minLength": 1,
+		    "examples": ["SSVC Supplier","Coordinator Triage", "SSVC Deployer"]
```

Comment:
I don't see those in https://github.com/CERTCC/SSVC/tree/main/data/json/decision_points. Shouldn't that be something like `Safety Impact`, `Report Public`, `Technical Impact`?

Reply:
These are Decision Point Groups - which targets grouping a set of distinct Decision Points to make sure they are captured according to a `role` - so related Decision Points are bundled together. For e.g., `Technical Impact` is a Decision Point itself , the examples of `SSVC Supplier` is here
https://github.com/CERTCC/SSVC/blob/3fd228b2cc83f59e94f7a00bf773233c7671e8c7/src/ssvc/dp_groups/ssvc/supplier.py#L60

Reply:
My bad. I didn't realize that this was in the Decision Point Group file.